### ISSUE TITLE
Cannot score teams that visited no checkpoints e.g. due to dropping out

### ISSUE THREAD

**Issue by danielswaine** [danielswaine/SodburyChallenge#6]
A team drops out before starting so therefore there wil be no times associated with that team and also no checkpoints. If you try to publish without anything written in the visited checkpoints, you cannot publish results...

> I think we should update the whole time logging form to make it more obvious what the fields do:
> - Start time field with a 'dropped out' checkbox next to it. When checkbox is selected the time field should be disabled and actual_start_time should be set to nil upon commit. The checkboxes on the subsequent time fields should also become automatically selected.
> - Phone in time field with forgot checkbox next to it. Selecting checkbox disables field, sets phone_in_time to nil on commit, but doesn't affect the other time fields.
> - Finish time field with 'didn't finish' checkbox next to it which disables finish time field and sets finish_time to nil on commit.
> 
> We could improve the status page by replacing predicted time with N/A for times that have been force set to nil because of drop outs etc.
> 
> I'll make a separate issue for that.
